pkg/tests/tasks/security/authorization/deny_test.go

```diff
@@ -48,7 +48,7 @@ func TestAuthorizationDenyAllow(t *testing.T) {
 			oc.ApplyString(t, ns, DenyGETPolicy)
 
 			t.LogStep("Verify that GET request is denied")
-			assertRequestDenied(t, ns, httpbinRequest("GET", "/get"))
+			assertRequestDenied(t, ns, httpbinRequest("GET", "/get"), "403", "Got the expected 403 Forbidden response")
```

Comment:
Passing in the entire success message is not that great. It's better if this function accepts only the httpStatusCode and then formats the success message on its own (IMHO the message doesn't really need to say "Forbidden"... a simple `fmt.Sprintf("Got the expected %s response code", httpStatusCode)` is fine).